postgres/application.yaml

```diff
@@ -1,17 +1,13 @@
-apiVersion: apps.redskyops.dev/v1alpha1
+apiVersion: apps.optimize.stormforge.io/v1alpha1
 kind: Application
 metadata:
   name: postgres
-
 resources:
 - postgres.yaml
-
 parameters:
 - containerResources:
     selector: >-
-      app.kubernetes.io/instance=postgres-stormforge-example,
-      app.kubernetes.io/name=postgres
-
+      app.kubernetes.io/instance=postgres-stormforge-example, app.kubernetes.io/name=postgres
```

Comment:
@jgustie Is this removed line break to be trusted?

Reply:
Yes, the `>-` indicates a "folded-strip" block scalar: so the interpretation is that new lines should be replaced by spaces and the trailing new line should not be preserved.